etc/environment-workflow.yml

```diff
@@ -1,4 +1,4 @@
-name: xcube_geodb
+name: xcube-geodb
```

Comment:
will this change where the conda package will be stored? 
https://anaconda.org/conda-forge/xcube_geodb 
I think it might be a good idea keeping the name as it was?

Reply:
No it's not. It's just an internal name.